src/hooks/useSubmitForm/useSubmitForm.ts

```diff
@@ -25,9 +27,14 @@ export const useSubmitForm = (activity: Activity): UseFormActivityHook => {
       },
     }
 
+    infoLog({ msg: 'Submit Form Response', response, activity })
```

Comment:
I would avoid including the full response in the log as it could contain PII/PHI.